README.md

```diff
@@ -1,9 +1,8 @@
-# [AudioVideoPlayBack ](https://appstore.home.mendix.com/index.html?v=7#1400594764694) : HTML5/Flash Audio & Video for [Mendix](http://mendix.com/)
+# Audio Video Playback : HTML5/Flash Audio & Video for [Mendix 7](http://mendix.com/)
```

Comment:
Why did u remove the link to mendix? Since you changed this to Mendix 7 does this means it does not work for other  Mendix version. I feel it should still remain Mendix irrespective of version?